### DISCUSSION TITLE
Search issues/MRs returns results from an author that was excluded from the search

### DISCUSSION THREAD

**Discussion by Woodz** [community/community#116903] [Projects and Issues]
### Select Topic Area

Bug

### Body

https://github.com/prettier/prettier/issues?q=typescript+parameter+decorator+comment+-author%3Adependabot+-author%3Aapp%2Frenovate+-author%3Aapp%2Fdependabot returns results authored by `app/renovate` (22 out of 25 on the first page) even though I excluded this author in my search (which reduced the number of results, so appears to be partially working)
![image](https://github.com/community/community/assets/966157/c6d5c56b-b261-4d0f-86a8-a72cbb0fdd5d)

**Comment by look** [comment]
Based on [my investigation into your other post](https://github.com/orgs/community/discussions/116904#discussioncomment-9023804), I believe that the correct name for bots is with a `[bot]` suffix, like `renovate[bot]`.
